Resources/Prototypes/SS220/Entities/Clothing/OuterClothing/coats.yml

```
@@ -215,3 +215,15 @@
     sprite: SS220/Clothing/OuterClothing/Coats/muscian_danger_coat.rsi
   - type: Clothing
     sprite: SS220/Clothing/OuterClothing/Coats/muscian_danger_coat.rsi
+
+# ГСБ
+- type: entity
```

```suggestion
- type: entity
```
В описании сказано, что он бронированный, но тут ни одного резиста не прописано?

---

блииин я такая тупая класс

---

А не, не тупая я.
У него в паренте же прописано

---

**Вы потревожили древнее зло!**
Апрува не будет